sbteclipse-core/src/main/scala/com/typesafe/sbteclipse/core/EclipsePlugin.scala

```diff
@@ -225,7 +225,7 @@ trait EclipsePlugin {
 
   object EclipseProjectFlavor extends Enumeration {
 
-    val Scala = Value
+    val ScalaIDE = Value
```

Comment:
Again, you're right this is ScalaIDE, but it'd be ideal not to break binary compatibility.

Can we keep the name Scala and have is synonymous?  I'd rather not break builds that already use this configuration.